config/crd/kustomization.yaml

```diff
@@ -5,17 +5,20 @@ resources:
 - bases/api.astarte-platform.org_astartes.yaml
 - bases/api.astarte-platform.org_astartevoyageringresses.yaml
 - bases/api.astarte-platform.org_flows.yaml
+- bases/ingress.astarte-platform.org_astartedefaultingresses.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
 - patches/webhook_in_astartes.yaml
 - patches/webhook_in_astartevoyageringresses.yaml
 - patches/webhook_in_flows.yaml
+#- patches/webhook_in_astartedefaultingresses.yaml
 # +kubebuilder:scaffold:crdkustomizewebhookpatch
 
 - patches/cainjection_in_astartes.yaml
 - patches/cainjection_in_astartevoyageringresses.yaml
 - patches/cainjection_in_flows.yaml
+#- patches/cainjection_in_astartedefaultingresses.yaml
```

Comment:
same as above

Reply:
The observation is correct for this line: it is needed for mutating webhooks, and we did scaffolded one for the ADI.